docs/reference/installation/interfaces.cue

```diff
@@ -1,46 +1,8 @@
 package metadata
 
 installation: {
-	#Commands: {
-		{[Name=string]: string | null}
-	} & {
-		_config_path: string | *null
-		let ConfigPath = _config_path
-
-		_shell: string | *null
-		let Shell = _shell
-
-		configure: string | null | *"none"
-		install:   string | null
-		logs:      string | null
-		reload:    string | null
-		restart:   string | null
-		start:     string | null
-		stop:      string | null
-		top:       string | null | *"vector top"
-		uninstall: string
-		upgrade:   string | null
-
-		if Shell == "bash" {
-			configure: string | *#"""
-					cat <<-'VECTORCFG' > \#(ConfigPath)
-					{config}
-					VECTORCFG
-					"""#
-		}
-
-		if Shell == "powershell" {
-			configure: string | *#"""
-					@"
-					{config}
-					"@ | Out-File -FilePath \#(ConfigPath)
-					"""#
-		}
-	}
-
 	#Interface: {
-		_shell: string | *null
-		let Shell = _shell
+		_shell: string
```

Comment:
Dropped the default from `_shell` and `configure`  to make sure we don't inadvertently hit this again.